packages/contracts/package.json

```diff
@@ -56,6 +55,7 @@
   },
   "dependencies": {
     "@gnosis.pm/safe-contracts": "https://github.com/safe-global/safe-contracts.git#767ef36bba88bdbc0c9fe3708a4290cabef4c376",
+    "@gnosis.pm/safe-contracts-1.4.1": "https://github.com/safe-global/safe-contracts.git#192c7dc67290940fcbc75165522bb86a37187069",
```

Comment:
Gnosis Safe v1.4.1 needs to be a production dependency because we import its `SafeProxy` in the `SphinxGnosisSafeProxyFactory` so that we can deploy the proxy

Reply:
I don't think either of these need to be production dependencies because we will not be compiling them in production, nor will we be importing any artifacts from them in production. Instead, we must once compile the `SphinxGnosisSafeProxyFactory` contract and store the compiler artifact in the `contracts` package. We'll then use that artifact whenever we need to deploy or interact with this contract.